test/cypress/integration/payment-links/create-payment-link.cy.test.js

```diff
@@ -204,6 +204,33 @@ describe('The create payment link flow', () => {
         cy.get('button').should('exist').should('contain', 'Create payment link')
       })
 
+      it('should show reporting column component', () => {
+        cy.get('a#add-reporting-column').should('exist').should('contain', 'Add a reporting column')
+        cy.get('a#add-reporting-column').click()
+        cy.location().should((location) => expect(location.pathname).to.eq('/create-payment-link/add-reporting-column'))
+      })
+
+      it('should reject empty values and accept valid input', () => {
+        cy.get('button#submit-reporting-column').click()
+        cy.location().should((location) => expect(location.pathname).to.eq('/create-payment-link/add-reporting-column'))
+        cy.get('div.govuk-error-summary').should('exist')
+        cy.get('input#metadata-column-header').type('key')
+        cy.get('input#metadata-cell-value').type('value')
+        cy.get('button#submit-reporting-column').click()
```

Comment:
Do we need some assertion here?

Reply:
There is an assertion on location directly following I believe